@xen-orchestra/web/src/pages/vm/[id]/system.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="system">
+    <div class="colum">
+      <UiCard><VmGeneralInfo :vm /></UiCard>
+      <UiCard><VmSystemNetworking :vm /></UiCard>
+      <UiCard><VmStorageConfig :vm /></UiCard>
+      <UiCard><VmResource :vm /></UiCard>
```

Comment:
Prefer to use `UiCard` component in the component itself instead
```suggestion
      <VmGeneralInfo :vm />
      <VmSystemNetworking :vm />
      <VmStorageConfig :vm />
      <VmResource :vm />
```